README.md

```diff
@@ -45,12 +45,27 @@ var theMeta = {
     modernizr: {group: 'Plugins', type: 'boolean', description: 'Whether or not to include modernizr on the page'},
     framework: {name: 'Framework', group: 'Plugins', type: 'options', options: ['None', {text:'AngularJS', value: 'angular'}, {text:'Backbone.js', value: 'backbone'}], description: 'Whether to include any additional framework'},
     iAmReadOnly: { name: 'I am read only', type: 'label', description: 'Label types use a label tag for read-only properties', showHelp: false }
-    
+
+};
+
+// This is the customTypes object that describes additionnal types, and their renderers (optional)
+var theCustomTypes = {
+	ref: {
+        name: 'ref', // name of custom type (required)
```

Comment:
nowhere. I guess it's legacy from first version, before I used property name. I've corrected that